lib/features/composer/presentation/composer_controller.dart

```diff
@@ -139,6 +139,8 @@ class ComposerController extends BaseController {
   final ccEmailAddressController = TextEditingController();
   final bccEmailAddressController = TextEditingController();
   final searchIdentitiesInputController = TextEditingController();
+  final displayTextInsertLinkDialogController = TextEditingController();
+  final urlInsertLinkDialogController = TextEditingController();
```

Comment:
Why not include it in the dialog widget? It should be managed by dialog

Reply:
done